site.someones.Lonewolf.yml

```diff
@@ -0,0 +1,62 @@
+app-id: site.someones.Lonewolf
+runtime: org.gnome.Platform
+# Stuck with GNOME 42 until tauri supports newer webkitgtk
+runtime-version: '42'
```

Comment:
thing is this runtime is EOL. You'll need to build webkit or fix tauri.

Reply:
just to clarify. We don't accept new submissions with EOL runtime.

Build this with GNOME 45.